src/2d/CMakeLists.txt

```diff
@@ -2,12 +2,15 @@ fips_begin_module(ari_2d)
 	fips_vs_warning_level(3)
 	fips_deps(ari_en ari_gfx)
     fips_files(
+        FrameData2D.hpp		
+        Sprite.hpp				  Sprite.cpp
 		Camera2D.hpp              Camera2D.cpp		
-        FrameData2D.hpp			 
         Node2D.hpp
 		RenderSystem2D.hpp        RenderSystem2D.cpp
         SceneSystem2D.hpp         SceneSystem2D.cpp
-        Sprite.hpp                Sprite.cpp
+		CanvasNode.cpp			  CanvasNode.hpp
+		Layout.cpp				  Layout.hpp
+
```

Comment:
Sort the files by name.

Reply:
done. When or where should I use sx_rect instead of air::core::RectI ?

Reply:
done! and where or when should I use ari::core::RectI instead of sx_rect in general?

Reply:
Always

Reply:
I changed my mind, delete RectI completely and use sx_rect every where.

Reply:
ok sure

Reply:
Then should I change Rect in viewport.hpp to sx_rect type?

Reply:
yes

Reply:
applied!